popup.js

```diff
@@ -12,12 +12,31 @@ function sendRequest(data) {
 }
 
 jQuery(document).ready(function($) {
-    //send message request to content script
-    $('#btnUpload').click(function() {
-        sendRequest({
-            type: 'addJson',
-            jsontext: $('#jsontext').val()
-        });
+    $('#form-import').submit(function(e) {
```

Comment:
@jinhduong Form onsubmit is a more correct approach, for the simple reason that a form can also be submitted with the `<ENTER>` key, and not just by clicking the submit button. For example: http://codepen.io/juno_okyo/pen/yOjaEZ